### ISSUE TITLE
Demonstrate viability of crowdsourcing

### ISSUE THREAD

**Issue by leeper** [leeper/RL10N#6]
I will put together a template translation task for MTurk (and/or maybe CrowdFlower) and attempt to obtain translations as a demonstration of how this would work. There are two uses of crowdsourcing here:
1. Generating translations
2. Validating translations that we generate automatically with one of the translation APIs (#4)

For the first, we could proceed in two steps:
- [ ] Translate one of the existing package translations (e.g., a base package), because that gives us something to validate against (i.e., the existing translation)
- [ ] Translate something new that we could have manually validated

For the second, the process would be:
1. Generate machine translations
2. Place these into a crowdsourcing context for validation

Things that I need to work on:
- [ ] Generate a task template to be used for obtaining new translations
- [ ] Generate a task template to be used for validating machine translations
- [ ] Write up an MTurkR (or crowdflower) script to generate and retrieve translations and/or validations
  - [ ] This will probably require language-specific qualification tests
  - [ ] Build in "known answers" (MTurk) or "gold questions" (CrowdFlower) to automatically flag problems
  - [ ] Implement ReviewPolicies (MTurk) to obtain consensus validations
- [ ] Figure out appropriate pay for each task

**Comment by richierocks** [comment]
> There are two uses of crowdsourcing

Agreed.  I think that the second use case is likely to be the most popular, assuming we can make automated translations really easy.  So having task templates for validation is higher priority than for generating translations.

> Translate one of the existing package translations

I think this is the best idea.  How about the [C-level messages from nlme](https://svn.r-project.org/R-packages/trunk/nlme/po/fr.po)?  There are only 10 of them, they contain some nicely tricky language, and I have access to French, German, Korean, and Polish speakers here at work so we have an extra level of validation.

**Comment by leeper** [comment]
I agree. I'll use that example file and try to get this fielded sometime during the coming week.
